docs/en/user-manual/advanced_configuration.md

```diff
@@ -202,7 +202,7 @@ configuration file in the `directory_users` section:
 
 ```
 directory_users:
-  extension: extenstions_config.yml  # reference to file with custom mapping information
+  extension: extension_config.yml  # reference to file with custom mapping information
```

Comment:
Sorry to nitpick, but to maintain the consistency I mentioned, we should specify the file as `extension-config.yml` (note the dash `-` instead of the underscore)